code/game/objects/items/storage/pouch.dm

```diff
@@ -517,6 +517,25 @@
 /obj/item/storage/pouch/medical_injectors/firstaid
 	desc = "Standard marine first-aid injector pouch. Specialized to store only auto-injectors. Contains basic injectors, a stabilizing injector, stimulant injector, and an emergency injector."
 
+/obj/item/storage/pouch/medical_injectors/basic
+	name = "medical injector pouch"
+	desc = "Standard marine first-aid injector pouch. Specialized to store only auto-injectors. Contains the basic injectors and two stabilizing injectors along with room to store specialized injectors."
+	icon_state = "firstaid_injector"
+	sprite_slots = 5
```

Comment:
You don't need to have stuff like sprite slots or icon state or anything but the description since they're all established previously under /medical_injectors and thus inherited.